iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/ArmTreeUtils.java

```diff
@@ -132,4 +133,14 @@ public static <T extends ArmTree> Stream<T> findAllNodes(ArmTree tree, Class<T>
       .filter(type::isInstance)
       .map(type::cast));
   }
+
+  /**
+   * If the provided expression is a function call, return the function name, otherwise return null.
+   */
+  public static Identifier functionNameOrNull(Expression expression) {
```

Comment:
I feel like the method name doesn't convey the fact that it applies only to function calls (i.e. not to function declarations). Since it's a public utility method that could be used elsewhere, maybe rename it to something like `calledFunctionNameOrNull`?